<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Your Results</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0">
    <property name="margin">
     <number>1</number>
    </property>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Comments</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="headerText"/>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QTextBrowser" name="contents">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <item row="0" column="1" rowspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="headerBox">
         <property name="text">
          <string>Include Headers</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2" colspan="2">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="7" rowspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="6" rowspan="2">
        <widget class="QComboBox" name="delims">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>tab</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>comma</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>semicolon</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="5" rowspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Separator:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="8" rowspan="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Save Data</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="9">
        <widget class="Line" name="line_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="5" colspan="4">
        <widget class="QFrame" name="plotSaveFrame">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Width:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="imageWidthBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>4000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveImageButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Save image</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Generate Project Summary</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="QTextBrowser" name="comments">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>531</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveImageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>projectSummary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>533</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delims</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>delimiterChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>saveImage()</slot>
  <slot>projectSummary()</slot>
  <slot>delimiterChanged(int)</slot>
 </slots>
</ui>
